src/main/resources/application-form-flow-library-test.yaml

```
@@ -3,3 +3,5 @@ spring:
     baselineOnMigrate: true
     placeholders:
       uuid_function: "random_uuid"
+  messages:
+    basename: messages,messages-form-flow
```

Nitpick, add a space after the comma.
```suggestion
    basename: messages, messages-form-flow
```